src/topology/unit_test.rs

```diff
@@ -70,21 +103,43 @@ impl UnitTest {
         );
 
         for check in &self.checks {
-            if let Some(results) = results.get(&check.extract_from) {
-                let mut failed_conditions = Vec::new();
-                for (index, cond) in check.conditions.iter().enumerate() {
-                    if results.iter().find(|e| cond.check(e)).is_none() {
-                        failed_conditions.push(index);
-                    }
+            if let Some((inputs, outputs)) = results.get(&check.extract_from) {
+                if check.conditions.is_empty() {
+                    inspections.push(format!(
+                        "check transform '{}' payloads (JSON encoded):\n{}\n{}",
```

Comment:
What is printed under `payloads` seems to be not JSON encoded though? It seems to be two objects separated by a newline (which is not valid a JSON)

Reply:
We could clarify here by making it more wordy, to something like `payloads (each event encoded as JSON):`, but in my mind it's pretty clear.

```
check transform 'thing' payloads (JSON encoded):
  input: {"message":"{\"hello\":\"world\"}","timestamp":"2020-02-07T15:46:34.771387075Z"}
  output: {"hello":"world","timestamp":"2020-02-07T15:46:34.771387075Z"}
```

Reply:
I've changed it to `events encoded as JSON`, could still be misinterpreted though. Whatcha think?